src/main.cpp

```diff
@@ -345,6 +345,7 @@ static void write_loop(FrameWriterParams params)
             if (params.enable_audio)
             {
                 pulseParams.audio_frame_size = frame_writer->get_audio_buffer_size();
+		pulseParams.sample_rate = params.sample_rate;
```

Comment:
This has wrong indentation.

Reply:
Yes, the coding style is a mess. My text editor kept using the wrong indentation, and only in the process of trying to change this to contribute to this project did I later realise it had a function to change the coding style used by the automatic indenter, and that this project had a completely different coding style than I am used to in other aspects as well. I am still in the process of cleaning all that up. 

...It seems my editor used tabs instead of spaces here, though, which looks exactly the same in my editor, but not in GitHub. So thank you for this tip too; I would be a lot less likely to catch it without it.